### DISCUSSION TITLE
Private container registries on AWS cloud

### DISCUSSION THREAD

**Discussion by manuelesimi** [nextflow-io/nextflow#2143] [Q&A] [answered]
Hi,

It doesn't seem that Nextflow natively supports connection to private registries. What is the suggested solution to access to private images from AWS Batch?

Thanks,
manuele

**Comment by pditommaso** [comment, marked as answer]
Use AWS ECR which access is controlled via IAM perms

**Reply by manuelesimi** [reply]
Sounds good. Thanks for clarifying.
